From a20cb3872c02241e4f0f7cc26933a43bac7d1cbb Mon Sep 17 00:00:00 2001
From: Xiao Ni <xni@redhat.com>
Date: Thu, 18 Apr 2024 18:23:20 +0800
Subject: [PATCH 61/66] tests/01r5integ.broken

01r5integ can be run successfully 152 times without error with
kernel 6.9.0-rc4 and mdadm - v4.3-51-g52bead95. So remove this
one broken case.

Signed-off-by: Xiao Ni <xni@redhat.com>
Signed-off-by: Mariusz Tkaczyk <mariusz.tkaczyk@linux.intel.com>
---
 tests/01r5integ.broken | 7 -------
 1 file changed, 7 deletions(-)
 delete mode 100644 tests/01r5integ.broken

diff --git a/tests/01r5integ.broken b/tests/01r5integ.broken
deleted file mode 100644
index 20737637..00000000
--- a/tests/01r5integ.broken
+++ /dev/null
@@ -1,7 +0,0 @@
-fails rarely
-
-Fails about 1 in every 30 runs with a sha mismatch error:
-
-    c49ab26e1b01def7874af9b8a6d6d0c29fdfafe6 /dev/md0 does not match
-    15dc2f73262f811ada53c65e505ceec9cf025cb9 /dev/md0 with /dev/loop3
-    missing
-- 
2.41.0

